README.md

```diff
@@ -2,7 +2,43 @@
 
 This repository is a collection of useful scripts and tools for TAs and professors to manage students workloads.
 
-## get_url.py
+## Cronjobs
+
+### ope-notebook-culler
+
+This cronjob runs once every 24 hours at 7am, removing all notebooks & pvcs from the rhods-notebooks namespace. To add resources to the rhods-notebooks namespace:
+
+1. Ensure you are logged in to your OpenShift account via the CLI and you have access to rhods-notebooks namespace.
+2. Switch to rhods-notebooks namespace:
+```
+	oc project rhods-notebooks
+```
+
+3. From cronjobs/ope-notebook-culler/ directory run:
+```
+	oc apply -k .
+```
+
+	This will deploy all the necessary resources for the cronjob to run on the specified schedule.
+
+Alternatively, to run the script immediately: 
+
+1. Ensure you followed the steps above
+2. Verify the cronjob ope-notebook-culler exists
+```
+	oc get cronjob ope-notebook-culler
```

Comment:
The new cronjob name is nb-culler. So the comand should be changed to`oc get cronjob nb-culler`